terms.html

```diff
@@ -96,6 +97,17 @@
 A role an <a>entity</a> may perform by asserting <a>claims</a> about one or more
 <a>subjects</a>, creating a <a>verifiable credential</a> from these claims,
 and transmitting the verifiable credential to a <a>holder</a>.
+  </dd>
+  <dt><dfn data-lt="presentation">presentation</dfn></dt>
+  <dd>
+Data derived from one or more credentials, fitted to the needs of sharing
+with a particular verifier, and encoded in such a way that the verifier may
+trust authorship of the data after a process of cryptographic verification. A
+credential itself may be presented, in which case the credential becomes a
```

Comment:
The following three sentences are problematic. Can we strike them entirely? Over-specifying things will make it more difficult for us to support broader use cases in the future.

Reply:
Can you give me the first few words and the last few words of the sentences you're talking about? I'm not sure what "the following 3 sentences" means because of the way github is showing me the map between comment and lines...